SparkPost/2web2ui
# FE-804: Add VWO script to head tag

### What Changed
 - Added script tag from https://app.vwo.com/#/settings/code?token=eyJhY2NvdW50X2lkIjo0NjE1NjUsInR5cGUiOiJ0cmFja2luZ0NvZGUiLCJ2ZXJzaW9uIjoxLCJoYXNoIjoiMjFiYzQ3NTU0MTcxOTcwMzE3MDRlZmRlMWZiZjllZDQifQ%3D%3D&integrationType=vwo-async to our head tag in the app

### How To Test
 - Verify with Smartcode Checker on above page
 - Interact with the app and see requests made to https://dev.visualwebsiteoptimizer.com/settings.js
